### DISCUSSION TITLE
Avoid PHP-FPM Reloading in Apache

### DISCUSSION THREAD

**Discussion by luckyshot** [deployphp/deployer#3065] [Help needed]
The documentation explains how to fix the caching/reloading issue with PHP-FPM for Nginx and Caddy:

https://deployer.org/docs/7.x/avoid-php-fpm-reloading

How do we fix the caching issue in Apache?

Thank you.

**Comment by AlecDr** [comment]
Hi @luckyshot, 

did you manage to find a solution without reloading php-fpm?

**Reply by luckyshot** [reply]
Hi Alec, 

I got it working eventually but not sure how exactly.  I changed way too many settings and even moved to a new server. Completely forgot about this issue since it got no replies. 

Oh, and I'm using Deployer v6.8.0, couldn't upgrade my code to v7 since documentation is scarce and unclear, I've wasted too many hours trying to upgrade and finally decided to stay on 6.8.0 until I eventually move everything to a CI/CD pipeline.

Sorry buddy

**Reply by AlecDr** [reply]
Thank's for the response Xavi!
